diagrams/sequenceDiagrams.md

```diff
@@ -0,0 +1,38 @@
+## A user requesting posts
+```mermaid
+sequenceDiagram
+participant w as web page (index.html)
+participant r as React Router (app.js)
+participant rfc as React Feed Componant (feed.js)
+participant exr as express app router
+participant exc as express controller (controllers/posts.js)
+participant mg as mongoose posts model (/models/posts.js)
+participant tmg as mongoose token model (/models/token_generator.js)
+participant mdb as MongoDb
+
+
+note left of w: Flow of time  ⇓ ⇓ ⇓ 
+
+
+w->>+r: Tell "/posts" is loaded
+note right of w: React
+rect rgb(0, 4, 255)
+
+
+r->>+rfc: React controller routes to the feed componant
```

Comment:
small repeated typo - quick find and replace all should fix: componant should be component ;)